tests/unit/storage_v3_property_store.cpp

```diff
@@ -0,0 +1,629 @@
+// Copyright 2022 Memgraph Ltd.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt; by using this file, you agree to be bound by the terms of the Business Source
+// License, and you may not use this file except in compliance with the Business Source License.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
+
+#include <limits>
+
+#include "storage/v3/property_store.hpp"
+#include "storage/v3/property_value.hpp"
+#include "storage/v3/temporal.hpp"
+
+namespace memgraph::storage::v3::tests {
+
+using testing::UnorderedElementsAre;
+
+const PropertyValue kSampleValues[] = {
```

Comment:
This will ve visible in our namespace now and the name is kind of generic, maybe add it as static variable inside fixture, where you can also instantitae `PropertyStore`?